postgkyl/data/read_gkyl_adios.py

```diff
@@ -34,7 +34,7 @@ def _is_compatible(self) -> bool:
     # imported when actially needed
     try:
       import adios2
-      fh = adios2.open(self.file_name, "rra")
+      fh = adios2.open(self.file_name, 'rra')
```

Comment:
@pcagas 
I assume we don't want to support "TimeMesh" and "Data", without a number after these names, right? (i.e. We only want to support TimeMesh0/Data0, TimeMesh1/Data1, etc)

Because presently gkyl is setup to produce such files but maybe it thouldn't (?). Maybe that's legacy and we should remove it from gkyl